package.json

```diff
@@ -235,6 +235,7 @@
     "@metamask-institutional/rpc-allowlist": "^1.0.0",
     "@metamask-institutional/sdk": "^0.1.20",
     "@metamask-institutional/transaction-update": "^0.1.30",
+    "@metamask/accounts-controller": "git://github.com/metamask/core/packages/accounts-controller",
```

Comment:
Is this to access unreleased changes during development? Just curious!

Reply:
Yes, it is just for development, it will be removed when the pull request the ready for review.